docs/source/tutorials/fashion_mnist.ipynb

```diff
@@ -0,0 +1,1257 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `find_issues` method can automatically infer the types of issues to be checked for based on the provided arguments. Here, we provide `features` and `pred_probs` as arguments. If you want to check for a specific issue type, you can do so using the `issue_types` argument. Check the [documentation](https://docs.cleanlab.ai/stable/cleanlab/datalab/datalab.html#cleanlab.datalab.datalab.Datalab.find_issues) for more comprehensive guide on `find_issues` method."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "lab.find_issues(features=features, pred_probs=pred_probs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### View report\n",
+    "\n",
+    "After the audit is complete, we can view the results and information regarding the issues using `report` method."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "lab.report(include_description=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Label issues\n",
+    "\n",
+    "In this section, we focus on visualizing mislabeled examples in the dataset. Label errors occur when the given label for an image is incorrect, usually due to mislabeling during dataset creation. Some of these examples are available at [labelerrors.com](https://labelerrors.com/) under the Caltech-256 dataset.\n",
+    "\n",
+    "`get_issues` method returns all the information related to the issue specified. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "label_issues = lab.get_issues(\"label\")\n",
+    "label_issues.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The above dataframe contains a `label_score` for each example. These numeric scores lie between 0 and 1, where lower scores indicate examples more likely to be mislabeled. It contains a boolean column `is_label_issue` specifying whether or not each example is identified to have a label issue (indicating it is likely mislabeled).\n",
+    "\n",
+    "We filter the `label_issues` DataFrame to find examples that are considered to have label errors and sort by `label_score` in ascending order to get the most severe mislabeled examples."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "label_issues_df = label_issues.query(\"is_label_issue\").sort_values(\"label_score\")\n",
+    "label_issues_df.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<details><summary>See the implementation of `plot_label_issue_examples` **(click to expand)**</summary>\n",
```

Comment:
```suggestion
    "<details><summary>We define a helper method `plot_label_issue_examples` to visualize results **(click to expand)**</summary>\n",
```

Reply:
you shouldn't lead with: "See the implementation of..." when you haven't mentioned this method at all yet at this point in the notebook